src/gui/advancedsettings.cpp

```diff
@@ -466,6 +466,7 @@ void AdvancedSettings::loadAdvancedSettings()
         // Tested in Qt 5.6.0. For more info see:
         // https://github.com/qbittorrent/qBittorrent/issues/5131
         // https://github.com/qbittorrent/qBittorrent/pull/5135
+        // TODO: remove this line after we require Qt >= 5.7.1 (see qt bug report)
```

Comment:
5.9 is minimum supported Qt version currently.

Reply:
in master IIRC

Reply:
Of course.
This PR is for master branch.
@sledgehammer999, glad to see you again.

Reply:
On qbittorrent.org the Windows 32-bit release is listed as using Qt 5.6.3.

Reply:
>On qbittorrent.org the Windows 32-bit release is listed as using Qt 5.6.3.

It is on v4_1_x branch and it doesn't matter.
See #10047 and https://github.com/qbittorrent/qBittorrent/blob/master/INSTALL#L14

Reply:
I'll move this into a separate PR